frontend/Dockerfile

```diff
@@ -8,9 +8,10 @@ RUN npm run build
 
 ### STAGE 2: Run ###
 FROM nginx:stable-alpine
+RUN apk --no-cache add openssl
 COPY .docker/nginx.conf /etc/nginx/conf.d/default.conf
 RUN mkdir -p /etc/nginx/ssl
-COPY ssl/server* /etc/nginx/ssl/
+RUN openssl req -x509 -newkey rsa:4096 -keyout /etc/nginx/ssl/server.key -out /etc/nginx/ssl/server.cert -days 365 -subj '/CN=pwndoc' -nodes
```

Comment:
If using this then a new certificate would be issued at each build. You would have to add the exception in your browser each time you re-build the app.
Same thing if you want to use a custom certificate signed by a CA.
Checking for certificate existence before generating a new one would solve that